thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/cube/cost/OeRatioCostFunction.java

```diff
@@ -0,0 +1,103 @@
+package org.apache.pinot.thirdeye.cube.cost;
+
+import com.google.common.base.Preconditions;
+import com.google.common.math.DoubleMath;
+import java.util.Map;
+import org.apache.pinot.thirdeye.cube.data.cube.CubeUtils;
+
+
+public class OeRatioCostFunction implements CostFunction {
```

Comment:
Could you add some documents for this class? Is this cost function works for all ratio metrics or it only works for OE ratio?

Reply:
Renamed to `RatioCostFunction` and added JavaDoc.